cv19ResSupportV3/V4/Helpers/PredicateHelpers.cs

```diff
@@ -0,0 +1,10 @@
+namespace cv19ResSupportV3.V4.Helpers
+{
+    public static class Predicates
+    {
+        public static bool IsNotNullAndNotEmpty(string testString)
+        {
+            return testString != null && testString.Trim() != "";
+        }
+    }
+}
```

Comment:
Do we need to define this separately?
We could just use the ```String.IsNullOrEmpty(s)``` method whenever we want to check if string is null or empty

Reply:
I remember reading something about the IsNullOrEmpty before, and people were suggesting not to use it as it depends on your environment (essentially side effects). So I wrote a custom one, which also ads the flexibility of extending the definition of empty _(maybe add check for unicode whitespaces or something of the sort)_ 😁

Reply:
Agreed to leave this for later, as this needs looking into.